### PR TITLE
Fixed bugs in query

### PR DESCRIPTION
**Pull request by hvlxh**
*title*

### CODE REVIEW

File: src/query/Query.js [hunk 1]
```diff
@@ -90,7 +90,7 @@ class Query {
 		if (type === 0x09) {
 			Frog.eventEmitter.emit("queryHandshakePacket", { query: { info: this.info, socket: this.socket }, type, magic, msg, remoteInfo });
 
-			Logger.info(getKey("query.server.network.packets.handshake").replace("%s%", `${remoteInfo.address}`));
+			Logger.debug(getKey("query.server.network.packets.handshake").replace("%s%", `${remoteInfo.address}`));
```
Comment (andriycraft) [inline src/query/Query.js:93]: revert

File: src/query/Query.js [hunk 2]
```diff
@@ -79,7 +79,7 @@ class Query {
 		}
 
 		if ((this.clientTokens.has(remoteInfo.address) && this.clientTokens.get(remoteInfo.address).expiresAt < Date.now()) || !this.clientTokens.has(remoteInfo.address)) {
-			Logger.info(getKey("query.server.network.generatingToken").replace("%s%", `${remoteInfo.address}`));
+			Logger.debug(getKey("query.server.network.generatingToken").replace("%s%", `${remoteInfo.address}`));
```
Comment (andriycraft) [inline src/query/Query.js:82]: also revert

File: test/Query.js [hunk 3]
```diff
@@ -15,9 +15,13 @@
...
+async function test() {
 		console.info("Basic data:", await util.queryBasic("0.0.0.0", 19133, { timeout: 1000 * 10 }));
 		console.info("Full data:", await util.queryFull("0.0.0.0", 19133, { timeout: 1000 * 10 }));
-	},
-};
+}
+
+module.exports = { test };
+
+if(process.argv) {
```
Comment (andriycraft) [inline test/Query.js:25]: revert

**Comment by hvlxh** [comment]
@andriycraft no one need to know about info so its should be debug message

**Comment by andriycraft** [comment]
i accidentally approved it

**Comment by andriycraft** [comment]
> @andriycraft no one need to know about info so its should be debug message

no

**Comment by andriycraft** [comment]
you can add something like `config.query.logPackets` or smth in config

File: src/Server.js [hunk 4]
```diff
@@ -128,7 +128,7 @@ module.exports = {
 		Frog.eventEmitter.emit("serverStart", {});
 
 		await assert(parseInt(config.performance.garbageCollectorDelay), NaN);
-		await assert(parseInt(config.world.tickSpeed), NaN);
+		await assert(parseInt(config.world.randomTickSpeed), NaN);
```
Comment (andriycraft) [inline src/Server.js:131]: revert. this will break the software

**Review by andriycraft — CHANGES_REQUESTED** [review]

**Review by andriycraft — APPROVED** [review]

**Comment by andriycraft** [comment]
i'll merge this later

**Pull request merged**